dotnet/src/Microsoft.AutoGen/Extensions/Anthropic/DTO/ChatCompletionRequest.cs

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Just FYI there's a library called `AutoSDK` which allows you to generate the client library from openapi.yaml. That might worth a try if you don't want to manually port the DTO classes.

- https://github.com/tryAGI/AutoSDK

Reply:
Ah, I just moved them over the AutoGen.Anthropic project so we can reuse the same code. Happy to rebuild the inner client on top of AutoSDK, if you think it is worth it?

Reply:
Yes I think we can look into that option (or any options that enables generating OpenAPI client from scheme) after this PR.